### PR TITLE
Add doc issue template

### PR DESCRIPTION
**Pull request by khushi-411**
Hi, Team!
The PR Fixes: https://github.com/cupy/cupy/issues/6293#issue-1094716054

cc: @kmaehashi @leofang @toslunar

### CODE REVIEW

**Comment by kmaehashi** [comment]
Thanks, basically LGTM!
Could you fix errors? https://github.com/cupy/cupy/blob/06ad27b7a3cd2d6a5a7fa8dd43062bc1e9319df4/.github/ISSUE_TEMPLATE/documentation.yml

**Comment by khushi-411** [comment]
Hi, @kmaehashi!
I made the required fixes. Could you please view it?

Here's the link to the file: https://github.com/cupy/cupy/blob/8d45e64baa2a7880bf7e690423eec023beaee277/.github/ISSUE_TEMPLATE/documentation.yml

Thanks (:

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+labels: ["cat:document"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this documentation request!
+  - type: textarea
+    id: description
+    attributes:
+      label: The doc issue
```
Comment (kmaehashi) [inline .github/ISSUE_TEMPLATE/documentation.yml:12]:
```suggestion
      label: Description
```

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      description: |
+        Please give a clear description of the doc issue. Make sure to provide relevant links to the document you are referring to.
```
Comment (kmaehashi) [inline .github/ISSUE_TEMPLATE/documentation.yml:14]:
```suggestion
        Please describe the issue with the current documentation. Make sure to provide relevant links to the document you are referring to.
```

**Review by kmaehashi — CHANGES_REQUESTED** [review]
Thanks!

**Comment by khushi-411** [comment]
Thanks a lot for reviewing, @kmaehashi! :)

**Pull request merged**